OneCommunityGlobal/HighestGoodNetworkApp
# Quick Fix for Links Not Opening in New Tab

# Description
Netlify Link doesn't allow client side routing. This PR fixes the same

## Related PRS (if any):
N/A

## Main changes explained:
Added _redirects so that client side routing is enabled

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Login and check if Cmd (Ctrl) + Click opens the link successfully.
